projects/storefrontapp-e2e-cypress/cypress/integration/regression/multisite-isolation/multisite-isolation.core-e2e-spec.ts

```diff
@@ -1,109 +1,50 @@
-
-      cy.cxConfig({
-        context: {
-          baseSite: [ELECTRONICS_STANDALONE_BASESITE],
-          currency: [ELECTRONICS_CURRENCY],
-        },
-      });
-    });
+      isolation.setBaseSiteConfig(ELECTRONICS_BASESITE);
 
-    it.skip('should register customer on standalone environment', () => {
-      cy.visit('/login/register');
-      register(user);
+      visitHomePage();
     });
 
-    it.skip('should not authenticate already registered customer outside isolated baseSite', () => {
-      Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
-
-      cy.cxConfig({
-        context: {
-          baseSite: [ELECTRONICS_BASESITE],
-          currency: [ELECTRONICS_CURRENCY],
-        },
-      });
-
-      cy.visit('/login');
-
-      login.listenForTokenAuthenticationRequest();
-      login.loginUser();
+    it('should revoke a session on baseSite to the isolated one switch', () => {
+      // 1. Login as an newly registered user in electronics-spa (on non-isolated)
+      isolation.verifyUserSession();
 
-      cy.wait('@tokenAuthentication')
-        .its('response.statusCode')
-        .should('eq', 400);
+      // 2. Keep the session and change url into electronics-standalone (isolated)
+      isolation.setBaseSiteConfig(ELECTRONICS_STANDALONE_BASESITE);
 
-      cy.get(login.userGreetSelector).should('not.exist');
+      // 3. The session should be revoked
+      visitHomePage();
 
       alerts
-        .getErrorAlert()
-        .should('contain', 'Bad credentials. Please login again');
+        .getSuccessAlert()
+        .should('contain', 'You have successfully signed out');
     });
+  });
 
-    it.skip('should not authenticate already registered customer outside isolated baseSite (B2B)', () => {
-      Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
-
-      cy.cxConfig({
-        context: {
-          baseSite: [POWERTOOLS_BASESITE],
-          currency: ['USD'],
-        },
-      });
-
-      cy.visit('/login');
-
-      login.listenForTokenAuthenticationRequest();
-      login.loginUser();
-
-      cy.wait('@tokenAuthentication')
-        .its('response.statusCode')
-        .should('eq', 400);
-
-      cy.get(login.userGreetSelector).should('not.exist');
+  describe('Standalone authorization', () => {
+    beforeEach(() => {
+      isolation.setBaseSiteConfig(ELECTRONICS_STANDALONE_BASESITE);
 
-      alerts
-        .getErrorAlert()
-        .should('contain', 'Bad credentials. Please login again');
+      visitHomePage();
     });
 
-    it.skip('should authenticate the customer on the isolated baseSite', () => {
-      cy.visit('/login');
-
-      login.listenForTokenAuthenticationRequest();
-      login.loginUser();
-
-      cy.wait('@tokenAuthentication')
-        .its('response.statusCode')
-        .should('eq', 200);
-
-      cy.get(login.userGreetSelector).should('exist');
+    it('should authenticate the customer on the isolated baseSite', () => {
+      isolation.verifyUserSession();
     });
```

Comment:
I see that the test cases have been reorganized. Can we have some verification that an account from one basesite cannot be used to log into another? It looks like we have dropped this scenario.